### DISCUSSION TITLE
OAuth2 with Ember?

### DISCUSSION THREAD

**Discussion by yatesco** [http4s/http4s#5722] [Q&A]
Hiya - https://github.com/http4s/http4s/issues/570 seems to suggest things have stalled. Is this on the roadmap or is there any prior art available?

Thanks!

**Comment by ChristopherDavenport** [comment]
I believe tsec, which I am working to get a minimal working version of operation again has oauth2 support integrated, and if it doesn't I think it has a solid baseline to work from with a little less of that complex machinery that the library utilizes.

https://github.com/jmcardon/tsec/tree/master/oauth2
